pcdsdevices/signal.py

```diff
@@ -920,13 +920,13 @@ class _OptionalEpicsSignal(Signal):
     """
 
     def __init__(self, read_pv, write_pv=None, *, name, parent=None, kind=None,
-                 **kwargs):
+                 attr_name=None, **kwargs):
```

Comment:
While the ophyd docstrings say this is optional, I think its default value should really be and empty string.  Looking at the mro, there's no `None` handling that saves us.

In that case, I think we shouldn't be re-defining this init kwarg, and should leave it as it was in `ophyd.Signal` (`attr_name=""`)